.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
             git checkout -qf FETCH_HEAD
           fi
 
-      - run: if [ -n \"$INFLUXDB_VERSION\" ] ; then ./install_influxdb.sh \"$INFLUXDB_VERSION\"; fi
+      - run: if [ -n "$INFLUXDB_VERSION" ] ; then ./install_influxdb.sh "$INFLUXDB_VERSION"; fi
```

Comment:
do we actually need the if? the env var is hardcoded to always be there anyway..

Reply:
good point